packages/domain-picker/test/use-your-domain-item.tsx

```diff
@@ -7,29 +7,28 @@
  */
 import * as React from 'react';
 import { render, screen, fireEvent } from '@testing-library/react';
-import '@testing-library/jest-dom/extend-expect';
 
 /**
  * Internal dependencies
  */
 import UseYourDomainItem from '../src/domain-picker/use-your-domain-item';
 
 describe( '<UseYourDomainItem />', () => {
-	test( 'Component renders with correct messaging', () => {
-		const onClick = jest.fn();
-
-		render( <UseYourDomainItem onClick={ onClick } /> );
+	it( 'should render with correct messaging', () => {
+		render( <UseYourDomainItem onClick={ jest.fn() } /> );
 
-		expect( screen.getByText( 'Already own a domain?' ) ).toBeTruthy();
-		expect( screen.getByText( "You can use it as your site's address." ) ).toBeTruthy();
+		expect( screen.getByText( 'Already own a domain?' ) ).toBeInTheDocument();
+		expect( screen.getByText( "You can use it as your site's address." ) ).toBeInTheDocument();
 	} );
 
-	test( 'Component fires onClick callback onClick', () => {
+	it( 'should fire onClick callback when clicked', () => {
 		const onClick = jest.fn();
 
 		render( <UseYourDomainItem onClick={ onClick } /> );
 
 		fireEvent.click( screen.getByText( 'Use a domain I own' ) );
-		expect( onClick ).toHaveBeenCalled();
+
+		// TODO: why is it called twice?
+		expect( onClick ).toHaveBeenCalledTimes( 2 );
```

Comment:
Change back to 1 once https://github.com/Automattic/wp-calypso/pull/51074 is merged